consensus/hotstuff/notifications/telemetry.go

```diff
@@ -215,10 +251,9 @@ type PathHandler struct {
 
 // NewPathHandler instantiate a new PathHandler.
 // The PathHandler has no currently open path
-func NewPathHandler(log zerolog.Logger, chain flow.ChainID) *PathHandler {
+func NewPathHandler(log zerolog.Logger) *PathHandler {
 	return &PathHandler{
-		chain:       chain,
-		log:         log.With().Str("hotstuff", "telemetry").Str("chain", chain.String()).Logger(),
+		log:         log.With().Str("hotstuff", "telemetry").Logger(),
```

Comment:
I personally would prefer to keep `chain.String()` as part of the log message. Maybe the keyword "chain" is misleading. 

Context:
* We have multiple consensus instances running in parallel: main consensus instance as well as several collector clusters. 
* The telemetry's main purpose is to allow analyzing the collective "swarm behaviour" of a particular consensus instance. For collector nodes, it is not apparent which collectors nodes belong to each of the clusters. 

The purpose of `chain` was to provide an easy way to filter for the telemetry logs of a particular cluster. For example the following [query](https://dapperlabs.grafana.net/explore?left=%7B%22datasource%22:%22grafanacloud-logs%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bnetwork%3D%5C%22mainnet20%5C%22%7D%20%7C%3D%20%5C%22telemetry%5C%22%20%7C%3D%20%5C%22cluster-58-bf8cea8d86557909a71c8113860e7d78e7c87c084140c35f3c6d6c07d29473ca%5C%22%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22grafanacloud-logs%22%7D,%22editorMode%22:%22code%22%7D%5D,%22range%22:%7B%22from%22:%22now-15m%22,%22to%22:%22now%22%7D%7D&orgId=1) collects the telemetry from one particular cluster: 
```
{network="mainnet20"} |= "telemetry" |= "cluster-58-bf8cea8d86557909a71c8113860e7d78e7c87c084140c35f3c6d6c07d29473ca"
```